### ISSUE TITLE
Strange MySQL error generated from using Resque Scheduler

### ISSUE THREAD

**Issue by willnance** [resque/resque-scheduler#215]
Hey, So I've been troubleshooting this for about a week now and have been hesitant to start an issue but I've followed instructions from a number of tutorials, particularly this tutorial on redis website:
 http://blog.togo.io/how-to/resque-scheduler/

which builds off of the cookie monster app which they build here:
http://blog.togo.io/news/resque-with-redis-to-go/

I believe I have everything set up the correct way in my app and I have followed the instructions multiple times and keep creating the exact same error, described here in detail:
http://stackoverflow.com/questions/15858776/mysql-error-using-resque-with-rails-table-does-not-exist

That SO post also has the resque.rb / rake files in there.

The only thing I can think of is that something might be off because I'm creating a delayed job with a after_create callback and that's messing something up. Note I'm not loading any YAML schedule because I'm only using delayed jobs, not scheduled jobs.

Basically, the app raises a MySQL exception because it's trying to read from a table that doesnt exist, but the table that it's looking for makes no sense. its looking for the table 'id' in my database, which obviously doesn't exist. SQL is SHOW FULL FIELDS FROM `id`

I've posted my code here for convenience in case you want to review it, but obviously I'm down to work in a Q&A type troubleshooting. I admit I had a tough time understanding the documentation, so I'm afraid this could possibly have something to do with my configuration. The relevant code other than the Resque config files are jobs/scheduler_job and app/models/assignment.rb
https://github.com/willnance/doctor-self-study
